scripts/docs/sources2html.pl

```diff
@@ -1262,33 +1256,46 @@ sub get_example {
   my $data_type  = shift;
   my $param      = shift;
   my $species    = shift;
-  my $database   = shift;
+  my $data       = shift;
   my $hostname   = shift;
   my $is_somatic = shift;
 
   return '' if (!$data_type_example{$data_type});
 
-  my $sql_query = ($is_somatic) ? 'sql_som' : 'sql';
-  my $sql = $data_type_example{$data_type}{$sql_query};
+  my $example;
   my $url = $data_type_example{$data_type}{'url'};
 
-  my $sth = get_connection_and_query($database, $hostname, $sql, [$param]);
-
-  if ($sth) {
-    my @result  = $sth->fetchrow_array;
-    my $example = ($result[1] eq 'QTL') ? $result[2] : $result[0];
-    my $url;
-    if ($result[1] && $data_type_example{$data_type}{$result[1]}) {
-      $url = $data_type_example{$data_type}{$result[1]};
-    }
-    else {
-      $url = $data_type_example{$data_type}{'url'};
-    }
-    if ($example && $url) {
-      $data_type =~ s/_/ /g;
-      my $example_url = "/$species/$url$example";
-      return qq{<a href="$example_url" target="_blank" title="See a $data_type example"><img src="$internal_link" alt="Link"/></a>};
+  my $sql_query = ($is_somatic) ? 'sql_som' : 'sql';
+  my $sql = $data_type_example{$data_type}{$sql_query};
+  if ($sql) {
+    # Features from database
+    my $database  = $data;
+    my $sth = get_connection_and_query($database, $hostname, $sql, [$param]);
+    if ($sth) {
+      my @result  = $sth->fetchrow_array;
+      $example = ($result[1] eq 'QTL') ? $result[2] : $result[0];
+      if ($result[1] && $data_type_example{$data_type}{$result[1]}) {
+        $url = $data_type_example{$data_type}{$result[1]};
+      }
+      else {
+        $url = $data_type_example{$data_type}{'url'};
+      }
     }
+  } else {
+    # Features from VCF
+    my $file_full_path = $data;
+    my $source_name    = $param;
+
+    # create a variation feature string
+    $example = `bcftools query -f "%CHROM:%POS:%REF-%ALT\n" $file_full_path | head -n 1`;
+    $example =~ s/-/_/;
```

Comment:
Should we do this on one query?

```suggestion
    $example = `bcftools query -f "%CHROM:%POS:%REF\_%ALT\n" $file_full_path | head -n 1`;
```

Reply:
Hi @nakib103, I tried, but it didn't work for me, it tries to fetch `%REF_` instead of `%REF`... Does it work for you?

Reply:
yeah, I needed to escape `_` with a `\_`. I am running the script the same as you had in this PR but when I tested with `\_` it worked fine.